src/Compiler/pars.fsy

```diff
@@ -1727,7 +1727,9 @@ classDefnMembersAtLeastOne:
   | classDefnMember opt_seps classDefnMembers
      { match $1, $3 with
        | [ SynMemberDefn.Interface(members=Some []; range=m) ], nextMember :: _ ->
-           warning(IndentationProblem(FSComp.SR.lexfltTokenIsOffsideOfContextStartedEarlier(warningStringOfPos m.Start), nextMember.Range))
+           let strictIndentation = parseState.LexBuffer.SupportsFeature LanguageFeature.StrictIndentation
+           let warnF = if strictIndentation then errorR else warning 
```

Comment:
Do we need a different error message if we treat it as an error? - It's no longer "Possible incorrect indentation" but just "Incorrect indentation"?

Reply:
Oh, right! That would be much better indeed.

Reply:
On the other hand this text tries to help a user if there is an indeed a wrong non-entirely intentional indentation. There's a separate parser error about some unfinished node, while this one tries to guess the user's indent. I think I'll leave it as is for now and will think more when I start testing the new recovery in the IDE. I've added a note about it to my todo list.